src/bilder/images/edxapp/templates/lms_only.yml

```diff
@@ -0,0 +1,114 @@
+ACCOUNT_MICROFRONTEND_URL: null
+ACE_CHANNEL_DEFAULT_EMAIL: django_email
+############################
+# We aren't using Sailthru #
+############################
+# ACE_CHANNEL_SAILTHRU_API_KEY: ''
+# ACE_CHANNEL_SAILTHRU_API_SECRET: ''
+# ACE_CHANNEL_SAILTHRU_DEBUG: true
+# ACE_CHANNEL_SAILTHRU_TEMPLATE_NAME: null
+ACE_CHANNEL_TRANSACTIONAL_EMAIL: django_email
+ACE_ENABLED_CHANNELS:
+- django_email
+ACE_ENABLED_POLICIES:
+- bulk_email_optout
+ACE_ROUTING_KEY: edx.lms.core.default
+# ALTERNATE_WORKER_QUEUES: cms  # The code already has this default
+ANALYTICS_API_KEY: ''
+ANALYTICS_API_URL: http://localhost:18100
```

Comment:
?

Reply:
That's for edX analytics which we don't have deployed and don't have immediate plans of setting up.

Reply:
So should we keep that value or add it when it's needed?

Reply:
I guess we can comment it out for now.